src/main/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy.java

```diff
@@ -260,7 +262,11 @@ protected SpecificUsersAuthorizationStrategy newInstanceWithoutAuthentication(
             if (StringUtils.isBlank(userid)) {
                 throw new FormException("userid must be specified", "userid");
             }
-            
+            // TODO use Jenkins.getInstance().getSecurityRealm().getUserIdStrategy().equals(userid, ACL.SYSTEM.getPrincipal().toString())) once Jenkins 1.566+
+            if (userid.equals(ACL.SYSTEM.getPrincipal())) {
+                throw new FormException(Messages.SpecificUsersAuthorizationStrategy_userid_notSystem(), "userid");
+            }
+

```

Comment:
Note for me:
* "SYSTEM" is set for the password for SYSTEM: https://github.com/jenkinsci/jenkins/blob/jenkins-1.645/core/src/main/java/hudson/security/ACL.java#L167
* It is not actually registered for the authentication manager and authentication never succeeds (as far as I tested).